pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1778,10 +1778,11 @@ func (dr *DashboardServiceImpl) searchDashboardsThroughK8sRaw(ctx context.Contex
 	}
 
 	if query.Title != "" {
-		// allow wildcard search
-		request.Query = "*" + strings.ToLower(query.Title) + "*"
```

Comment:
This will break the legacy fallback search: https://github.com/grafana/grafana/blob/main/pkg/registry/apis/dashboard/legacysearcher/search_client.go#L54 (which is now being used in ops)